recipe/build.sh

```diff
@@ -20,6 +20,7 @@ cd ..
 mkdir -p build
 cd build
 
+export CUDA_HOME=/usr/local/cuda
```

Comment:
For some reason, script_env isn't accepted in the meta.yaml. I think because it has script specified. I'll still try and see this can be moved to meta.yaml.